core/profiles/standard/standard.install

```diff
@@ -157,6 +157,28 @@ function standard_install() {
       ),
       'is_new' => TRUE,
     ),
+    array(
+      'type' => 'card',
+      'name' => st('Card'),
+      'description' => st('Add a piece of content to be used for blocks and views. A card does not have a public path (hide path display). Example use-case: Place card content as "Existing content" block in a Layout.'),
+      'settings' => array(
+        'status_default' => NODE_PUBLISHED,
+        'node_preview' => TRUE,
+        'promote_enabled' => TRUE,
+        'promote_default' => FALSE,
+        'sticky_enabled' => FALSE,
+        'sticky_default' => FALSE,
+        'revision_enabled' => TRUE,
+        'revision_default' => FALSE,
+        'scheduling_enabled' => FALSE,
+        'show_submitted_info' => TRUE,
+        'comment_default' => COMMENT_NODE_CLOSED,
+        'node_submitted' => FALSE,
+        'node_user_picture' => TRUE,
```

Comment:
same for user picture, this might set to FALSE automatically once `show_submitted_info` is false.